core/server/services/auth/authorize.js

```diff
@@ -37,7 +37,17 @@ const authorize = {
         };
     },
 
-    authorizeAdminAPI: [session.ensureUser]
+    authorizeAdminAPI: [session.ensureUser],
```

Comment:
Also left in to keep the rebase simple. after merge i will open a PR deleteing this.